### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pgivel**

### CODE REVIEW

**Review by SThor — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Incremental game (working title)
 
-## Available Scripts
+This project is a work-in-progress incremental game played in the brower. You play as a mercenary slowly building up their empire.
```
Comment (gfaugere) [inline README.md:3]: Typo: played in the bro**w**ser

File: README.md [hunk 2]
```diff
@@ -37,32 +49,32 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+## Contribute
 
-### Making a Progressive Web App
+If you wish to contibute, you're welcome to do so :) I am mostly looking for technical help as well as gameplay design, but if you see anything that you can improve, feel free to open a PR. If you want to talk further about this project, please get in touch with me via mail on paulgivel[at]gmail.com.
```
Comment (gfaugere) [inline README.md:54]: Typo: if you wish to cont**r**ibute

File: README.md [hunk 3]
```diff
@@ -37,32 +49,32 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+## Contribute
 
-### Making a Progressive Web App
+If you wish to contibute, you're welcome to do so :) I am mostly looking for technical help as well as gameplay design, but if you see anything that you can improve, feel free to open a PR. If you want to talk further about this project, please get in touch with me via mail on paulgivel[at]gmail.com. 
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
+This repo is open to #hacktoberfest pull requests, and in this context my company [Genymobile](https://www.genymobile.com/) is sponsoring me to work on this project.
 
-### Advanced Configuration
+## License
```
Comment (gfaugere) [inline README.md:58]: Move full license to separate file?

**Comment by SThor** [comment]
It seems I closed this PR a little bit too quick. I'm trying to re-open it, and if I can't I'll open another one.

**Pull request merged**